plasmapy/physics/tests/test_distribution.py

```diff
@@ -0,0 +1,35 @@
+"""Tests for functions that uses Distribution functions."""
+
+import numpy as np
+import pytest
+from astropy import units as u
+
+
+from ...constants import (m_p, m_e, c, mu0, k_B, e, eps0, pi, e)
+from ..atomic import (ion_mass, charge_state)
+from ..distribution import (Maxwellian_1D)
+
+T_e = 30000*u.K;
+V_drift = 10*u.m/u.s;
```

Comment:
I haven't looked at the speed of this test, but if it's not too slow (it's an integral, it shouldn't be) this would be a perfect place to test a few parameter combinations via pytest fixtures. I'd be happy to add them!